src/main/java/Event.java

```diff
@@ -0,0 +1,15 @@
+public class Event extends Task {
+    protected String from;
+    protected String to;
```

Comment:
"from" and "to" might be ambiguous, well at least from my pov. Instead of time, I could think of it in terms of place.
Other than that, nice code 👍🏻 good to go.